_posts/2016-01-30-css-essentials.md

```diff
@@ -2,7 +2,7 @@
 layout: post
 title: CSS-essentials, Onefootball open source CSS project
 permalink: /onefootball-open-source-css-essentails
-tags: css, flex, onefootball, css-essentials, open source
+tags: [css, flex, onefootball, css-essentials, open-source]
```

Comment:
Small inconsistency: we have `SVG` but `css`, either we keep all the tags lower-cased or we decide to write acronyms in capital characters.